src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -5470,13 +5470,33 @@ class Goo<T>
 }
 ";
 
-            var compilation = CreateCompilation(source);
+            var compilation = CreateCompilation(source, parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp7));
             compilation.VerifyDiagnostics(
                 // (2,2): error CS0404: Cannot apply attribute class 'Goo<T>' because it is generic
                 // [Goo<int>]
                 Diagnostic(ErrorCode.ERR_AttributeCantBeGeneric, "Goo<int>").WithArguments("Goo<T>").WithLocation(2, 2));
         }
 
+        [Fact]
+        public void AttributeContainsGenericWithGenericAttributeFeature()
+        {
+            string source = @"
+[Goo<int>]
+class G
+{
+}
+class Goo<T>
+{
+}
+";
+
+            var compilation = CreateCompilation(source);
+            compilation.VerifyDiagnostics(
```

Comment:
Should we also have a LangVer error here? (the user needs to fix two things to make this work)

Reply:
done